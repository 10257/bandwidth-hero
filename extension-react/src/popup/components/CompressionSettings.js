import React from 'react'
import { Form } from 'semantic-ui-react'

export default () => {
  return (
    <Form>
      <Form.Input label="Webtask URL" type="url" />
    </Form>
  )
}
