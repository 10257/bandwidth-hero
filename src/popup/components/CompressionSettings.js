import React from 'react'
import { Input, Button } from 'semantic-ui-react'

export default ({
  proxyUrl,
  onChange,
  onReset
}: {
  proxyUrl: string,
  onChange: (event: Event, data: {}) => void,
  onReset: () => void
}) => {
  return (
    <Input
      label="URL"
      type="url"
      fluid
      value={proxyUrl}
      onChange={onChange}
      action={<Button icon="history" onClick={onReset} />}
    />
  )
}
