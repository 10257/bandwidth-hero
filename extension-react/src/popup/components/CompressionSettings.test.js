import React from 'react'
import CompressionSettings from './CompressionSettings'
import renderer from 'react-test-renderer'

it('renders compression settings correctly', () => {
  const tree = renderer.create(<CompressionSettings />).toJSON()
  expect(tree).toMatchSnapshot()
})
