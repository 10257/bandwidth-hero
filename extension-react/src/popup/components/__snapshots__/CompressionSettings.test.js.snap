// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders compression settings correctly 1`] = `
<form
  className="ui form"
>
  <div
    className="field"
  >
    <label
      htmlFor={undefined}
    >
      Webtask URL
    </label>
    <div
      className="ui input"
    >
      <input
        required={undefined}
        type="url"
      />
    </div>
  </div>
</form>
`;
